docs/For Users/Getting Started.md

```diff
@@ -1,167 +1,100 @@
+You can also download the binary directly from the official website [here](https://dl.nwjs.io/).
 
-## Write NW.js App
+## Write your first NW.js application
 
-### Example 1 - Hello World
+Create a `package.json` with a `name` and `main` property. The `main` property references the first page opened by NW.js. This can be an HTML or JavaScript file.
 
-This the basic example shows how to write an NW.js app.
+Here’s an example on how to use an HTMl file as the entry point:
 
-**Step 1.** Create `package.json`:
+`package.json`
 
 ```json
 {
-  "name": "helloworld",
-  "main": "index.html"
+  "name": "nw-demo",
+  "main": "./index.html"
 }
 ```
 
-`package.json` is the manifest file in your app. It is written in [JSON format](http://www.json.org/). The `main` field figures out the first page opened by the NW.js if referenced to an HTML file, `"index.html"` in this example. And the `name` field is the unique name used among NW.js apps. See [Manifest Format](../References/Manifest Format.md) for more details.
-
-!!! tip "Use JS File as Main"
-    You can set JS file as in `"main"` field as well, like `"main.js"`. Then the JS file will be loaded in the background page on start and no window is opened by default. Usually you can do some initialization and open the window manually later. For example,
-    ```javascript
-    // initialize your app
-    // and ...
-    nw.Window.open('index.html', {}, function(win) {});
-    ```
-
-**Step 2.** Create `index.html`:
+`index.html`
 
 ```html
 <!DOCTYPE html>
 <html>
   <head>
     <title>Hello World!</title>
   </head>
+  <script>
+    // Call a Node module
+    const os = require("os");
+    document.write("You are running on ", os.platform());
+
+    // Call the NW API
+    const nwWindow = nw.Window.get();
+    nwWindow.showDevTools();
+
+    // You can also use any third party Node modules installed by you
+  </script>
   <body>
     <h1>Hello World!</h1>
   </body>
 </html>
 ```
 
-This is the normal HTML file. You can use any web technologies supported by latest browsers.
-
-**Step 3.** Run your app
-
-```bash
-cd /path/to/your/app
-/path/to/nw .
-```
-
-`/path/to/nw` is the binary file of NW.js. On Windows, it's `nw.exe`; On Linux, it's `nw`; On Mac, it's `nwjs.app/Contents/MacOS/nwjs`.
-
-!!! tip "Drag &amp; Drop on Windows"
-    On Windows, you can drag the `folder containing package.json` to `nw.exe` to run your app.
-
-### Example 2 - Using NW.js APIs
-
-All NW.js APIs are loaded in `nw` object globally and can be used directly in JavaScript files. See [API References](../index.md#references) for full list of supported APIs.
+Here’s an example on how to use a JavaScript file as the entry point:
 
-This example shows how to create a native context menu in your NW.js app. You can create `index.html` with following content:
```

Comment:
The Menu example should be kept somewhere?

Reply:
There is already a Menu example in the API section - that's why I said to learn more check out the API reference.

Reply:
My idea for the docs is include a complete example on how to use each API component under the definitions (taking inspiration from `esbuild`'s [docs](https://esbuild.github.io/api/#build-api)).